foam_wrapper/foam_wrapper.py

```diff
@@ -0,0 +1,228 @@
+""" foam_wrapper module
+
+Wrapper module for OpenFOAM using pythonFlu -python interface
+NOTE: at a moment pythonFlu supports only up to 2.0.1 -version of OpenFOAM
+
+"""
+
+from Foam import ref, man
+from simphony.cuds.mesh import Mesh, Point, Face, Edge, Cell
+#import inspect
+#import copy
+#from Foam.OpenFOAM import fileName, word
+from collections import defaultdict
+#from Foam.finiteVolume import volScalarField
+
+
+class Foam_wrapper(object):
+
+    def __init__(self, model):
+        self.model = model
+
+    def has_common_vertex(self, facePoints0, facePoints1):
+        for point in facePoints0:
+                if point in facePoints1:
+                    return True
+        return False
+
+    def get_mesh(self, foamCaseName):
+        """ Reads given OpenFoam -mesh from "foamCaseName" -directory and returns SimPhony mesh
+        """
+        argv=["test", "-case", foamCaseName]
+        args = ref.setRootCase(len(argv), argv)
+        runTime = man.createTime(args)
+        foamMesh = man.createMesh(runTime)
+
+        foamPoints = foamMesh.points()
+        foamFaces = foamMesh.faces()
+        foamCells = foamMesh.cells()
+
+        simphonyMesh = Mesh()
+
+        foamLabelToUuid=[]
+ 
+        for i in range(foamPoints.size()):
+            point = foamPoints[i]
+
+            spoint = Point((point.x(),point.y(),point.z()))
+            simphonyMesh.add_point(spoint)
+            foamLabelToUuid.append(spoint.uuid)
+                 
+        print "Points added"
```

Comment:
suggestion: It is common to remove `print` statements from library code. Logging messages however are much appreciated.

The proposed pattern is to have at the top of each module after the imports a module level logger instance 
```python
import logging
logger = logging.getLogger(__name__)
```

And then inside the code use `logger.info(msg)` or the other logging functions to log messages and events